DevCycle/DVCClient.swift

```diff
@@ -74,6 +76,18 @@ public class DVCClient {
                     Log.debug("Config: \(config)", tags: ["setup"])
                 }
                 self.config?.userConfig = config
+                
+                if (self.checkIfEdgeDBEnabled(config: (self.config?.userConfig)!, enableEdgeDB: self.enableEdgeDB)) {
+                    if (!(user.isAnonymous ?? false)) {
+                        self.service?.saveEntity(user: user, completion: { data, response, error in
+                            if error != nil {
+                                Log.error("Error saving user entity")
```

Comment:
Should the actual error be logged here?

Reply:
following the same pattern as JS SDK... We'll have the full error captured on the workers